src/browser.ts

```diff
@@ -69,7 +74,16 @@ export class GrpcClient {
 
   async createStub(service: protobuf.Service, opts: ClientStubOptions) {
     const authHeader = await this.auth.getRequestHeaders();
-    async function serviceClientImpl(method, requestData, callback) {
+
+    function serviceClientImpl(method, requestData, callback) {
+      const cancelController = new AbortController();
```

Comment:
I just realized that `AbortController` is not supported for all browsers (even though it works in all major ones). https://caniuse.com/#search=AbortController 

Just to stay on a safe side, let's add a check: `if (typeof window.AbortController === "undefined") { don't do anything }`

Reply:
Wouldn't it be a good idea to add some type of information on it not being supported? Or should it be completely silent. I changed it to do nothing for now.

Reply:
Let's think what happens when the `AbortController` is not supported. We definitely should not break the workflow - the regular request should still work. But if a cancellation is requested, we have two options: we can break (throw an exception), or we can ignore it. I suggest that if `AbortController` is not present, we just set `cancelController` to `undefined`; and in `cancel()`, we `console.warn()` that cancellation is not supported in this browser if it's `undefined`.

Reply:
I added a comment, tell me what you all think.

> 'AbortController not found: Cancellation is not supported in this environment'